### DISCUSSION TITLE
KusionStack + KCL 21th Weekly Meeting | 第 21 次周会 (2023.08.31)

### DISCUSSION THREAD

**Discussion by Peefy** [kcl-lang/community#6] [General]
会议方式
Peefy 邀请您参加腾讯会议
会议主题：KusionStack + KCL 社区双周会
会议时间：2023/08/03 20:00-20:45 (GMT+08:00) 中国标准时间 - 北京
重复周期：2023/08/03-2024/08/29 20:00-20:45, 每两周 (周四)

点击链接入会，或添加至会议列表：
https://meeting.tencent.com/dm/Hc6sNpqTWnPb

#腾讯会议：778-2381-6338

复制该信息，打开手机腾讯会议即可参与

**Comment by ArianEllison** [comment]
Bedankt voor het delen van de details. Dit ziet er goed georganiseerd en overzichtelijk uit. Het tweewekelijkse ritme maakt het makkelijk om de ontwikkelingen te volgen en vooruit te plannen. Ik vind het fijn dat KusionStack en KCL op één vaste plek samenkomen. Het tijdstip is ook ideaal voor mensen in China. Ik zal proberen deel te nemen aan deze sessie [casino Caspero](https://caspero-casino.nl/) en mee te luisteren. Het zou geweldig zijn om updates van de beheerders te horen en te ontdekken waar de community zich de komende tijd op richt. Hopelijk is er ruimte voor vragen en een open discussie. Ik kijk uit naar het gesprek en ben benieuwd hoe het project zich verder ontwikkelt.
